api/lib/routes/routeHelper.js

```diff
@@ -31,3 +31,20 @@ get any error during the schedule creation/update. */
       callback(error,{ 'statusCode':HttpStatus.INTERNAL_SERVER_ERROR });
     });
 }
+
+exports.deletePolicy = function(req, schedulerResponse, callback) {
```

Comment:
why do we pass in the schedulerResponse?!?

Same is true for createOrUpdatePolicy